charts/harmony-chart/Chart.yaml

```diff
@@ -26,23 +26,23 @@ dependencies:
   repository: https://charts.jetstack.io
```

Comment:
@gabor-boros you need to bump the version of the harmony chart to trigger a new release. The current version is `0.3.0`. This should be bumped to 0.4.0

Reply:
Bumped to 0.4.0